### PR TITLE
Add video library to the libraries

### PR DESCRIPTION
**Pull request by msayoung**

### CODE REVIEW

**Review by markconroy — COMMENTED** [review]
Is this video library used anywhere?

**Comment by msayoung** [comment]
It _should_ be being used here https://github.com/localgovdrupal/localgov_base/blob/1.x/templates/paragraphs/paragraph--localgov-video.html.twig but we forgot to define it

File: localgov_base.libraries.yml [hunk 1]
```diff
@@ -273,3 +273,9 @@ leaflet:
     theme:
       css/components/leaflet.ie11.css: {}
       css/components/leaflet.css: {}
+
+video:
+  css:
+    theme:
+      css/components/video.ie11.css: {}
```
Comment (markconroy) [inline localgov_base.libraries.yml]: Let's remove this ie11 line. We don't have any ie11 specific video css (I don't think).

After that, merge merge merge!

Comment (msayoung) [reply localgov_base.libraries.yml]: done. I'm hitting the merge button.
I guess we'll need a release at some point?

**Pull request merged**